src/nft/components/details/detailsV2/TabbedComponent.tsx

```diff
@@ -27,6 +25,23 @@ const Tab = styled(ThemedText.SubHeader)<{ isActive: boolean; numTabs: number }>
   }
 `
 
+const TabNumBubble = styled(ThemedText.UtilityBadge)`
+  background: ${({ theme }) => theme.backgroundOutline};
+  border-radius: 4px;
+  padding: 2px 4px;
+  color: ${({ theme }) => theme.textSecondary};
+  line-height: 12px;
+`
+
+export const TabTitleWithBubble = ({ title, bubbleNumber }: { title: React.ReactNode; bubbleNumber?: number }) => {
```

Comment:
I am wondering if this is the best way to do this - or if this should be the default tab component that is rendered and you can just add another prop to the `Tab` interface like `count` which is then passed into `bubbleNumber`

Reply:
The Titles are currently evenly split 3:3 on whether or not they include the count. Given that I have no strong preference but this solution does allow for flexibility

Reply:
for me this decision would be based around how tightly the title area is coupled to the overall tab component. if you can change the title very independently of the tab and have it render fine, i think it makes sense to have it here. if changing the tab component impacts title rendering significantly, i think having a `bubbleCount` as an optional tab prop will ensure that if someone messes with tab styling, they also have to update the title rendering

Reply:
Had a discussion with design. When the count is 0 we'll want to hide the bubble anyway so I've updated the Tab interface to have an optional `count` field and made the bubble with count the default. If that count is 0 or undefined, the bubble will be hidden.